app/assets/stylesheets/govuk_publishing_components/components/_contents-list.scss

```diff
@@ -76,6 +76,33 @@
   }
 }
 
+.gem-c-contents-list--alternate-active {
+  .gem-c-contents-list__list-item {
+    padding: govuk-spacing(2) 0;
+    margin: 0;
+
+    & span::before {
+      display: none;
+    }
+  }
+
+  .gem-c-contents-list__list-item--active {
+    padding: govuk-spacing(1) govuk-spacing(2);
+    margin: govuk-spacing(1) 0;
+    border-left-style: solid;
+    border-left-width: 4px;
+    border-color: govuk-colour("blue");
```

Comment:
Minor comment but perhaps these could be shortened with `border-left: 4px solid #1d70b8;`

Reply:
I've no idea why I did those so verbosely, I usually use shorthand! Now fixed, thanks.